lib/ansible/modules/cloud/azure/azure_rm_storageaccount.py

```diff
@@ -76,28 +76,28 @@
         version_added: "2.2"
     access_tier:
         description:
-            - The access tier for this storage account. Required for a storage account of kind 'BlobStorage'.
+            - The access tier for this storage account. Required for a storage account of I(kind=BlobStorage).
         choices:
             - Hot
             - Cool
         version_added: "2.4"
     force_delete_nonempty:
         description:
-            - Attempt deletion if resource already exists and cannot be updated
+            - Attempt deletion if resource already exists and cannot be updated.
         type: bool
         aliases:
             - force
     https_only:
         description:
-            -  Allows https traffic only to storage service if sets to true.
+            -  Allows https traffic only to storage service if sets to C(true).
         type: bool
         version_added: "2.8"
     blob_cors:
         description:
             - Specifies CORS rules for the Blob service.
             - You can include up to five CorsRule elements in the request.
-            - If no blob_cors elements are included in the argument list, nothing about CORS will be changed.
-            - "If you want to delete all CORS rules and disable CORS for the Blob service, explicitly set blob_cors: []."
+            - If no I(blob_cors) elements are included in the argument list, nothing about CORS will be changed.
```

Comment:
Same thing here - otherwise we would end up with I(....) everywhere.

```suggestion
            - If no blob_cors elements are included in the argument list, nothing about CORS will be changed.
```